src/common_args.rs

```diff
@@ -0,0 +1,51 @@
+use clap::Args;
+
+#[cfg(feature = "unstable")]
+use crate::config::CliConfig;
+
+#[derive(Debug, Clone, Args)]
+pub struct CommonArgs {
```

Comment:
In the future, things like `confirm_prompts` should be moved here as well.
But that would exceed the scope of this PR